modules/exploits/linux/http/seagate_central_ssh_user_add.rb

```diff
@@ -0,0 +1,215 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'net/http'
+require 'net/ssh'
+require 'net/ssh/command_stream'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::SSH
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Seagate Central Remote Code Execution Exploit",
```

Comment:
This looks more like an arbitrary user creation vulnerability (as per the file name).

Also the module name should describe the vulnerable device(s) rather than only the manufacturer name. Seagate make a lot of products.

Suggest something like this:

```suggestion
      'Name'           => "Seagate Central External NAS Arbitrary User Creation Exploit",
```

If you update the module name, the file name should also be updated to match.